### PR TITLE
sdv1.5(v2): textual inversion train and infer

### PR DESCRIPTION
**Pull request by wtomin**
- [x] Train and inference scripts of textual inversion with SDV1.5 and SDV2.0.
- [x] Detailed tutorial with two datasets as examples. Generated images are shown in the readme file.

### CODE REVIEW

File: examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+The standalone training command for SDv1.5 finetuning on the cat-toy dataset:
+```bash
+python train_textual_inversion.py \
+    --train_config configs/train/train_config_textual_inversion_v1.yaml  \
+    --output_path="output/"
+```
+
+Suppose the saved checkpoint file is `output/weights/SDv1.5_textual_inversion_3000_ti.ckpt`, we use the following command to run inference with the newly learned text embedding.
+```bash
+export MS_PYNATIVE_GE=1
```
Comment (zhtmike) [inline examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md:126]: is `MS_PYNATIVE_GE` necessary？

File: examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+## Preparation
+
+### Dependency
+Make sure the following frameworks are installed.
+
+- mindspore 2.1.0 (Ascend 910) / mindspore 2.2.1 (Ascend 910*)
+
+Enter the `example/stable_diffusion_v2` folder and run
+
+```shell l
```
Comment (zhtmike) [inline examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md:33]: typo? shell l

File: examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md (hunk 1 above)
Comment (wtomin) [reply examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md]: No, not necessary. Deleted.

File: examples/stable_diffusion_v2/ldm/modules/textual_inversion/text_encoder.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+# CLIP model
+from transformers import CLIPTokenizer
+
+import mindspore as ms
+from mindspore.common.initializer import TruncatedNormal, initializer
+
+
+class FrozenCLIPEmbedder_CLIPTokenizer(FrozenCLIPEmbedder):
+    """
+    A wrapper over FrozenCLIPEmbedder that uses CLIPTokenizer as the tokenizer instead of Simple Tokenizer
```
[the remark this answers is not in the document]
Comment (wtomin) [reply examples/stable_diffusion_v2/ldm/modules/textual_inversion/text_encoder.py]: Thanks for reminding me of this. I have merged mine with your code.

File: examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md (hunk 2 above)
Comment (wtomin) [reply examples/stable_diffusion_v2/docs/en/textual_inversion_finetune.md]: Typo. Fixed.

**Review by zhtmike — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]
nice

**Review by hadipash — APPROVED** [review]

**Pull request merged**